PublicMapping/districtbuilder-classic
# Make the existing tests run using scripts/test

## Overview

This PR makes the existing `redistricting/tests.py` run. It fixes enough Django and database problems to make the tests run, but doesn't fix the errors in the tests themselves. 

This also calls changes the STRTA `script/test` to run these tests to make it easier to set up CI. 

### Checklist


### Notes

The test db that Django stands up already has a row in the `django_site` table with "example.com" as the domain. Since this was causing a collision with the fixtures file, the fixtures file domain was changed to "examples.com". 

 A Django 1.8 issue made hardcoding the language necessary for testing. Refer to [this issue](https://github.com/django-parler/django-parler/issues/90).

## Testing Instructions

 * `vagrant ssh`
 * `./scripts/test`
 * Confirm that the tests run (not that they pass; they don't).
